cardano-launcher/app/Main.hs

```diff
@@ -147,14 +146,11 @@ main = silence $ do
             throwM . WorkingDirectoryDoesNotExist $ workingDir
 
         -- Configuration from the launcher options.
-        let configurationOptions :: ConfigurationOptions
-            configurationOptions = loConfiguration launcherOptions
+        let mConfigurationOptions :: Maybe ConfigurationOptions
+            mConfigurationOptions = loConfiguration launcherOptions
 
-        let walletPath :: WalletPath
-            walletPath = getWPath launcherOptions
-
-        let walletArgs :: WalletArguments
```

Comment:
the initial plan, was to just not have any args for daedalus in the config file
`daedalusBin` was purely the path to the binary, and nothing more
so i had remove the args support, rather then rename it all

then 2 days later, i needed some `daedalusArgs` to `--inspect-brk=5858` for debug purposes

maybe it could be re-added, as `daedalusArgs` now?

Reply:
Can we leave it as is for now?